DOL.WHD.Section14c.Web/src/modules/components/sectionEmployer/sectionEmployerTemplate.html

```diff
@@ -453,11 +453,11 @@
         </fieldset>
     </div>
     <div class="form-question-block" ng-class="validate('employer.temporaryAuthority') ? 'usa-input-error' : ''">
-        <div class="form-question-text">Is this a request for Temporary Authority?
+        <div class="form-question-text">
+            Is this a request for Temporary Authority by a vocational rehabilitation program administered by a State agency or the U.S. Veterans Administration?
             <helplink></helplink>
-            <helptext>Answer yes only if the application is being filed by a vocational rehabilitation program administered by a State agency or the U.S. Veterans Administration. See 29 C.F.R. &sect; 525.8.</helptext>
+            <helptext>Select “Yes” only if the application is being filed by a vocational rehabilitation program administered by a State agency or the U.S. Veterans Administration. See 29 C.F.R. § 525.8.</helptext>
```

Comment:
We can fix it later if it's an issue, but if the browser isn't interpreting the HTML as UTF-*, will it render the curly quotes (or whatever they're called) and section symbol correctly aside from using HTML entities?  We'll find out!  😄